src/containers/WindowSideBarCanvasPanel.js

```diff
@@ -31,13 +31,32 @@ const mapDispatchToProps = { setCanvas: actions.setCanvas };
  * @returns {label: {paddingLeft: number}}}
  */
 const styles = theme => ({
+  formControl: {
+    marginBottom: theme.spacing.unit,
+    marginTop: theme.spacing.unit,
+    minWidth: 120,
+  },
   label: {
     paddingLeft: theme.spacing.unit,
   },
   listItem: {
     borderBottom: '0.5px solid rgba(0,0,0,0.12)',
     paddingRight: theme.spacing.unit,
   },
+  select: {
+    '&:focus': {
+      backgroundColor: theme.palette.background.paper,
+      borderTopLeftRadius: theme.shape.borderRadius,
+      borderTopRightRadius: theme.shape.borderRadius,
+    },
+  },
+  selectEmpty: {
+    backgroundColor: theme.palette.background.paper,
+    borderTopLeftRadius: theme.shape.borderRadius,
+    borderTopRightRadius: theme.shape.borderRadius,
+    marginTop: theme.spacing.unit * 2,
```

Comment:
I think we'll also need to do something responsive when the companion window is in the bottom position; right now it ends up looking pretty funny with a huge top margin.

Reply:
I wonder, too, if some of this top margin belongs on the titleControls div in the companion window?